abis/sigma/abi.h

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Please check if managarm/qword ABIs are identical here. If yes, move the file to abis/mlibc and change managarm's and qword's symlinks.

Reply:
Yep they are thesame